components/library/SDL2/Makefile

```diff
@@ -13,12 +13,14 @@
 # Copyright (c) 2020 Andreas Wacknitz
 #
 
+BUILD_BITS= 64_and_32
 USE_PARALLEL_BUILD= yes
```

Comment:
Please add a comment why 32 bit is needed. Otherwise it is prone to be removed again.